h2o-py/tests/testdir_algos/automl/pyunit_automl_args.py

```diff
@@ -417,31 +429,32 @@ def test_default_max_runtime_if_no_max_models_provided():
 
 
 pu.run_tests([
-    test_invalid_project_name,
-    test_early_stopping_defaults,
-    test_early_stopping_args,
-    test_no_x_train_set_only,
-    test_no_x_train_and_validation_sets,
-    test_no_x_train_and_test_sets,
-    test_no_x_train_and_validation_and_test_sets,
-    test_no_x_y_as_idx_train_and_validation_and_test_sets,
-    test_predict_on_train_set,
-    test_nfolds_param,
-    test_nfolds_eq_0,
-    test_fold_column,
-    test_weights_column,
-    test_fold_column_with_weights_column,
-    test_balance_classes,
-    test_nfolds_default_and_fold_assignements_skipped_by_default,
-    test_keep_cross_validation_fold_assignment_enabled_with_nfolds_neq_0,
-    test_keep_cross_validation_fold_assignment_enabled_with_nfolds_eq_0,
-    test_automl_stops_after_max_models,
-    test_stacked_ensembles_are_trained_after_max_models,
-    test_stacked_ensembles_are_trained_with_blending_frame_even_if_nfolds_eq_0,
-    test_frames_can_be_passed_as_key,
-    test_no_time_limit_if_max_models_is_provided,
-    test_max_runtime_secs_alone,
-    test_max_runtime_secs_can_be_set_in_combination_with_max_models_and_max_models_wins,
-    test_max_runtime_secs_can_be_set_in_combination_with_max_models_and_max_runtime_wins,
-    test_default_max_runtime_if_no_max_models_provided,
+    # test_invalid_project_name,
+    # test_early_stopping_defaults,
+    # test_early_stopping_args,
+    # test_no_x_train_set_only,
+    # test_no_x_train_and_validation_sets,
+    # test_no_x_train_and_test_sets,
+    # test_no_x_train_and_validation_and_test_sets,
+    # test_no_x_y_as_idx_train_and_validation_and_test_sets,
+    # test_predict_on_train_set,
+    # test_nfolds_param,
+    # test_nfolds_eq_0,
+    # test_fold_column,
+    # test_weights_column,
+    # test_fold_column_with_weights_column,
+    # test_balance_classes,
+    # test_nfolds_default_and_fold_assignements_skipped_by_default,
+    # test_keep_cross_validation_fold_assignment_enabled_with_nfolds_neq_0,
+    # test_keep_cross_validation_fold_assignment_enabled_with_nfolds_eq_0,
+    # test_automl_stops_after_max_models,
+    # test_stacked_ensembles_are_trained_after_max_models,
+    # test_stacked_ensembles_are_trained_with_blending_frame_even_if_nfolds_eq_0,
+    # test_frames_can_be_passed_as_key,
+    # test_no_time_limit_if_max_models_is_provided,
+    test_optional_SEs_trained_by_default_when_no_time_limit,
+    # test_max_runtime_secs_alone,
+    # test_max_runtime_secs_can_be_set_in_combination_with_max_models_and_max_models_wins,
+    # test_max_runtime_secs_can_be_set_in_combination_with_max_models_and_max_runtime_wins,
+    # test_default_max_runtime_if_no_max_models_provided,
```

Comment:
These will get uncommented before merging, right?

Reply:
oups! 2nd time I do it on this PR I think